### PR TITLE
Container timeout calls only listener#timed_out, not listener#finished.

### PR DESCRIPTION
**Pull request by fhwang**

### CODE REVIEW

**Comment by pbrisbin** [comment]
Seems reasonable to me /cc @dblandin though

**Comment by fhwang** [comment]
@dblandin checks are passing, can you take a look?

File: lib/cc/analyzer/container.rb [hunk 1]
```diff
@@ -27,16 +27,14 @@ def initialize(
...
         @on_output = ->(*) { }
-
         @timed_out = false
         @stderr_io = StringIO.new
       end
 
-      def on_output(delimeter = "\n", &block)
-        @output_delimeter = delimeter
+      def on_output(delimiter = "\n", &block)
+        @output_delimiter = delimiter
```
Comment (pbrisbin) [inline lib/cc/analyzer/container.rb]: Nice catch, but what do you think about pulling this to its own PR? This logic is so critical and so futzy, I want to be super careful with any changes to it.

**Comment by dblandin** [comment]
Looks good to me! :+1:

File: lib/cc/analyzer/container.rb (hunk 1 above)
Comment (fhwang) [inline lib/cc/analyzer/container.rb]: Yeah, that's fair, I'll back it out.

**Pull request merged**